sos/plugins/collectd.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2016 Archit Sharma <archit.sh@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+import re
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Collectd(Plugin, RedHatPlugin):
+    '''
+    Collectd config collector
+    '''
+    plugin_name = "collectd"
+    profiles = ('services', 'webserver')
+
+    files = (
+        '/etc/collectd.conf',
+    )
+
+    def setup(self):
+        self.add_copy_spec([
+            '/etc/collectd.conf',
+            '/etc/collectd.d/*.conf',
+        ])
+
+        p = re.compile('^LoadPlugin.*')
+        with open("/etc/collectd.conf") as f:
+            for line in f:
+                if p.match(line):
+                    self.add_alert("Active Plugin found: %s <br>\n" %
```

Comment:
It would be much better if `collectd` had some command that could be run to obtain the plugin list; we try to avoid parsing configuration as much as possible.